README.md

```diff
@@ -310,73 +333,92 @@ The example requires two trading accounts: `buyer` and `seller`:
         --testnet \
         --yes
     ```
+   Check the `seller` account balance:
+
+    ```bash
+    $ provenanced q bank balances "$SELLER" \
+      --chain-id "$CHAIN_ID" \
+      --node "$NODE" \
+      --testnet
+    ```
 
 ### 5. Store and Instantiate the `ats-smart-contract`
 
-1. Copy the previously built `artifacts/ats-smart-contract.wasm` to the root directory of the Provenance git project:
+1. Copy the previously built `artifacts/ats_smart_contract.wasm` to the root directory of the Provenance git project:
 
     ```bash
-    $ cp ats-smart-contract/artifacts/ats-smart-contract.wasm "$PIO_HOME"
+    $ cp <<Your sc project dir>>/ats-smart-contract/artifacts/ats_smart_contract.wasm "$PIO_HOME"
     $ cd "$PIO_HOME"
     ```
 
 2. Deploy the `ats-smart-contract` to Provenance and store the resulting code ID:
 
     ```bash
-    $ store_result=$(provenanced tx wasm store "ats_smart_contract.wasm" \
+    $ provenanced tx wasm store "ats_smart_contract.wasm" \
         --from "$NODE0" \
+        --node "$NODE" \
         --home "$PIO_NODE" \
         --chain-id "$CHAIN_ID" \
         --keyring-backend test \
         --gas auto \
         --gas-prices 1905nhash \
-        --gas-adjustment 2 \
+        --gas-adjustment 1.2 \
         --testnet \
-        --yes)
-    $ export CODE_ID=$(jq '.logs[0].events[] | select(.type == "store_code").attributes[] | select(.key == "code_id").value | tonumber' <<< "$store_result")
```

Comment:
If this `jq` query isn't picking out the code ID anymore, I'd prefer if the query were were updated instead of having to manually pick out the code ID out each time, since there's quite a lot of output